blackbox/docs/sql/analyzer.txt

```diff
@@ -533,6 +533,16 @@ Converts alphabetic, numeric, and symbolic Unicode characters which are not in
 the first 127 ASCII characters (the "Basic Latin" Unicode block) into their
 ASCII equivalents, if one exists.
 
+.. _flattengraph-tokenfilter:
+
+``flatten_graph``
+-----------------
+
+``type='flatten_graph'``
```

Comment:
pls document this a bit more and add some links, i think for an end user this is not enough